### ISSUE TITLE
a11y audit: Tooltip a11y fixes

### ISSUE THREAD

**Issue by marijohannessen** [IBM/carbon-components-react#631]
### Detailed description

- [ ] The id __carbon-tooltip_t5u2uqglcl specified for WAI-ARIA property aria-owns on element div is not valid. See rule help for remediation guidance.
![screen shot 2018-02-19 at 10 28 07 am](https://user-images.githubusercontent.com/5447411/36387840-a7d533d2-155f-11e8-8d26-cfa6c9505ac7.png)
- [ ] The WAI-ARIA role(s) and/or attribute(s) aria-expanded are not valid for the element div. See rule help for guidance.
![screen shot 2018-02-19 at 10 31 02 am](https://user-images.githubusercontent.com/5447411/36387948-00985300-1560-11e8-8309-484a4376a1df.png)
- [ ] All content must reside within a WAI-ARIA landmark or labelled region role.
![screen shot 2018-02-19 at 10 32 36 am](https://user-images.githubusercontent.com/5447411/36388011-373bfbf0-1560-11e8-8893-fc99cf13fd6b.png)
- [ ] Normal tooltip: Cannot use keyboard interaction to tab to tooltip without icon
- [ ] Normal tooltip: Cannot use keyboard interaction to open the "click to open" tooltip
- [ ] A WAI-ARIA widget must have an accessible name (see message help text for details).
![screen shot 2018-02-19 at 10 37 43 am](https://user-images.githubusercontent.com/5447411/36388265-eea73bb0-1560-11e8-8659-79a6b5e4d2e8.png)
